.gitignore

```diff
@@ -2,33 +2,353 @@
 /build
 AssemblyInfo.cs
 
-# Standard VS.NET and ReSharper Foo
-.vs/*
-src/*/obj
-src/*/bin
-samples/*/obj
-samples/*/bin
-*.csproj.user
-*ReSharper.user
-_ReSharper*
-*.suo
 *.cache
```

Comment:
Not sure what writes a .cache file, do you know?

Reply:
Visual Studio, but it's always been inside the obj folder as far as I know, so this looks redundant.